cluster-up/kubectl.sh

```diff
@@ -30,7 +30,7 @@ source ${KUBEVIRTCI_PATH}/hack/common.sh
 source ${KUBEVIRTCI_CLUSTER_PATH}/$KUBEVIRT_PROVIDER/provider.sh
 source ${KUBEVIRTCI_PATH}/hack/config.sh
 
-if [ "$1" == "console" ] || [ "$1" == "vnc" ] || [ "$1" == "start" ] || [ "$1" == "stop" ] || [ "$1" == "migrate" ]; then
+if [ "$1" == "console" ] || [ "$1" == "vnc" ] || [ "$1" == "usbredir" ] || [ "$1" == "start" ] || [ "$1" == "stop" ] || [ "$1" == "migrate" ]; then
```

Comment:
This should be done in `kubevirt/kubevirtci`. The entire `cluster-up/` directory is imported from there.
https://github.com/kubevirt/kubevirtci/

Reply:
Ah, right. I've been using this locally for testing, didn't know it cames from kubevirtci, thanks!